src/domain/repository/secureAccessKeyCmdRepo.go

```diff
@@ -0,0 +1,11 @@
+package repository
+
+import (
+	"github.com/goinfinite/os/src/domain/dto"
+	"github.com/goinfinite/os/src/domain/valueObject"
+)
+
+type SecureAccessKeyCmdRepo interface {
```

Comment:
I don't think we need a query/cmd repo for keys. That could be on Account repo for sure. There is no plan to have keys for anything rather than accounts.

Reply:
I prefer to keep the infrastructure separate to make its maintenance more modular, similar to mappings and vhosts.